read_to_trie.py

```diff
@@ -0,0 +1,37 @@
+from trie import Trie
+import os
+import re
+
+pattern = r'[^a-zA-Z0-9\s]'
+
+
+def read_files(trie: Trie, dir_path: str, file_index: int = 0) -> None:
```

Comment:
You need to insert all the words to the data structure

Reply:
done